crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -474,7 +511,13 @@ private QueryExpression rewriteUpdateQueryForCompositeSearch(CompositeMetadata m
      * The implementation passes the request to the back-end.
      */
     @StopWatch(loggerName = "stopwatch.com.redhat.lightblue.mediator.Mediator", sizeCalculatorClass = "com.redhat.lightblue.mediator.ResponsePayloadSizeCalculator")
-    public Response find(FindRequest req) {
+    public Response find(FindRequest req, RequestMetrics metrics) {
```

Comment:
Did you take a stab at also monitoring the findAndStream version? It will have to be implemented a bit differently but it's possible. I can give an example but I have to run right now.